### DISCUSSION TITLE
TP-Link Archer C20 V5.0 supported

### DISCUSSION THREAD

**Discussion by DRCRecoveryData** [xchwarze/wifi-pineapple-cloner#3] [Show and tell]
Hi,

I have testing TP-Link Archer C20 V5.0 after upgrade 4mb rom to 16mb working with breedweb and can flash gl-inet mt300n v2 and wifipineapple rom too 🎉🎉

Can you add it to devices list?

![image](https://github.com/xchwarze/wifi-pineapple-cloner/assets/85211068/355abc2e-63ba-429b-831f-ac22187d349e)
![image](https://github.com/xchwarze/wifi-pineapple-cloner/assets/85211068/a0bf4307-5b12-4758-af42-8d1b88a7d1af)
![image](https://github.com/xchwarze/wifi-pineapple-cloner/assets/85211068/b2b6af57-ee18-4063-995c-04d4e8424a79)
![image](https://github.com/xchwarze/wifi-pineapple-cloner/assets/85211068/80859840-425e-48d5-b660-1b8aeaddfa69)

**Comment by xchwarze** [comment]
hello! but from what I see in the pictures that router does not meet the minimum requirements. It has no usb.
The intention of the listing is that they are mass consumption devices ready to use and also comply with everything we can find in the original hard.

**Reply by DRCRecoveryData** [reply]
> hello! but from what I see in the pictures that router does not meet the minimum requirements. It has no usb.
> 
> The intention of the listing is that they are mass consumption devices ready to use and also comply with everything we can find in the original hard.

Hi,

Rom had replace 4mb to 16mb with breedweb and the config match to mt300n-v2 🫡🫡, also usb host can modify on pcb same as archer v4 on openwrt forum
